package.json

```diff
@@ -62,54 +62,52 @@
     }
   },
   "devDependencies": {
-    "@aws-sdk/credential-providers": "^3.515.0",
+    "@aws-sdk/credential-providers": "^3.632.0",
     "@iarna/toml": "^2.2.5",
     "@istanbuljs/nyc-config-typescript": "^1.0.2",
-    "@microsoft/api-extractor": "^7.43.1",
-    "@microsoft/tsdoc-config": "^0.16.2",
+    "@microsoft/api-extractor": "^7.47.5",
+    "@microsoft/tsdoc-config": "^0.17.0",
     "@mongodb-js/zstd": "^1.2.0",
-    "@types/chai": "^4.3.14",
+    "@types/chai": "^4.3.17",
     "@types/chai-subset": "^1.3.5",
     "@types/express": "^4.17.21",
     "@types/kerberos": "^1.1.5",
-    "@types/mocha": "^10.0.6",
-    "@types/node": "^20.14.10",
+    "@types/mocha": "^10.0.7",
+    "@types/node": "^22.1.0",
     "@types/saslprep": "^1.0.3",
     "@types/semver": "^7.5.8",
     "@types/sinon": "^17.0.3",
     "@types/sinon-chai": "^3.2.12",
-    "@types/whatwg-url": "^11.0.4",
-    "@typescript-eslint/eslint-plugin": "^5.62.0",
-    "@typescript-eslint/parser": "^5.62.0",
+    "@types/whatwg-url": "^11.0.5",
+    "@typescript-eslint/eslint-plugin": "^8.0.0",
+    "@typescript-eslint/parser": "^8.0.0",
     "chai": "^4.4.1",
     "chai-subset": "^1.6.0",
     "chalk": "^4.1.2",
-    "eslint": "^8.56.0",
-    "eslint-config-prettier": "^8.10.0",
-    "eslint-plugin-github": "^4.10.2",
-    "eslint-plugin-import": "^2.29.1",
+    "eslint": "^9.0.0",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-mocha": "^10.4.1",
-    "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-simple-import-sort": "^10.0.0",
-    "eslint-plugin-tsdoc": "^0.2.17",
-    "eslint-plugin-unused-imports": "^2.0.0",
+    "eslint-plugin-prettier": "^5.2.1",
+    "eslint-plugin-simple-import-sort": "^12.1.1",
+    "eslint-plugin-tsdoc": "^0.3.0",
+    "eslint-plugin-unused-imports": "^4.0.0",
     "express": "^4.19.2",
     "gcp-metadata": "^5.3.0",
     "js-yaml": "^4.1.0",
     "mocha": "^10.4.0",
     "mocha-sinon": "^2.1.2",
-    "mongodb-client-encryption": "^6.1.0-alpha.0",
-    "mongodb-legacy": "^6.0.1",
+    "mongodb-client-encryption": "6.1.0-alpha.0",
```

Comment:
I had to pin this to the alpha, CI is red until https://github.com/mongodb/node-mongodb-native/pull/4189#pullrequestreview-2243087220 merges.